orttraining/orttraining/test/gradient/gradient_op_test_utils.cc

```diff
@@ -184,6 +230,7 @@ void GradientOpTester::Run(
       fetches_ = ExecuteModel<onnxruntime::training::TrainingSession>(*p_model, session_object, expect_result, expected_failure_string, run_options,
                                                                       feeds, output_names, provider_type);
     }
+    }
```

Comment:
Same comment as above about leftover debugging code.

Reply:
This is actually a result of bad code formatting the code should be indented differently.

Reply:
The indentation has been fixed it should be clear that both curly braces are part of the code.